mu-plugins/blocks/multisite-latest-posts/postcss/style.pcss

```diff
@@ -0,0 +1,45 @@
+
+body {
+	--wporg-link-title-color: var(--wp--preset--color--charcoal-1);
+	--wporg-link-title-font-family: var(--wp--preset--font-family--eb-garamond);
+	--wporg-link-title-font-size: var(--wp--preset--font-size--heading-5);
+	--wporg-link-title-font-size-line-height: var(--wp--custom--heading--level-3--typography--line-height);
+	--wporg-link-spacing: var(--wp--preset--spacing--20, 20px);
+	--wporg-post-spacing: var(--wp--preset--spacing--30, 30px);
+	--wporg-link-details-font-size: var(--wp--preset--font-size--small);
+}
```

Comment:
I'm torn on assigning this CSS up here... but it feels weird having those uber-specific variables littered in the code base. 

Doing it this way also seems like it could pollute the CSS variable space too though...

Reply:
Maybe style it roughly so it doesn't look bad on its own, and add the theme-specific styles into the parent theme?